install/kubernetes/helm/subcharts/mixer/values.yaml

```diff
@@ -18,11 +18,16 @@ telemetry:
   replicaCount: 1
   autoscaleEnabled: true
   autoscaleMin: 1
-  autoscaleMax: 5
+  autoscaleMax: 15
   cpu:
     targetAverageUtilization: 80
   sessionAffinityEnabled: false
 
+  # based on measurements 50ms p50 translates to p99 of under 1s. This is ok for telemetry which is inherently async.
```

Comment:
please explain why user may need to tweak this.  e.g. is this something we really want to expose